plugins/org.jkiss.dbeaver.registry/OSGI-INF/l10n/bundle.properties

```diff
@@ -16,18 +16,19 @@ extension-point.org.jkiss.dbeaver.task.name = Tasks and task types
 extension-point.org.jkiss.dbeaver.productFeature.name = Product feature
 
 language.en.label=English
-language.zh.label=Simplified chinese
-language.tw.label=Traditional chinese
-language.ru.label=Russian
-language.fr.label=French
-language.de.label=German
-language.es.label=Spanish
-language.pt_BR.label=Portuguese (BR)
-language.it.label=Italian
-language.ja.label=Japanese
-language.uk.label=Ukrainian
-language.ko.label=Korean
-language.ro.label=Romanian
+language.zh.label=\u7b80\u4f53\u4e2d\u6587
+language.tw.label=\u7e41\u9ad4\u4e2d\u6587
+language.ru.label=\u0420\u0443\u0441\u0441\u043a\u0438\u0439
+language.fr.label=Fran\u00e7ais
+language.de.label=Deutsch
+language.es.label=Espa\u00f1ol
+language.pt_BR.label=Portugu\u00eas (BR)
+language.it.label=Italiano
+language.ja.label=\u65e5\u672c\u8a9e
+language.uk.label=\u0423\u043a\u0440\u0430\u0457\u043d\u0441\u044c\u043a\u0430
+language.ko.label=\ud55c\uad6d\uc5b4
+language.ro.label=Rom\u00e2n\u0103
+language.ar.label=\u0627\u0644\u0639\u0631\u0628\u064a\u0629
```

Comment:
Maybe we should hardcode them in the `plugin.xml` and add a comment stating these shouldn't be translated?